RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@ Unreleased
 * [***] New Block: Search Block [#3210]
 * [**] The media upload options of the Image, Video and Gallery block automatically opens when the respective block is inserted. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2700]
 * Image block: Add a "featured" banner. (Android only) [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3371]
+* [*] Fixed a bug where the Search block was stealing focus from the Image block upon updating image asset [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3442]
```

Comment:
I was checking the [release notes of `1.52.0`](https://github.com/wordpress-mobile/gutenberg-mobile/releases/tag/v1.52.0) and I didn't see this change, looks like this got merged after the `1.52.0` code freeze so maybe we should move it to the `Unreleased` section, what do you think @AmandaRiu ?

Reply:
I think that makes sense. How would I go about doing this now? Should I open a PR?